### DISCUSSION TITLE
The fundamental region of FCC system

### DISCUSSION THREAD

**Discussion by M-Eddahbi** [mtex-toolbox/mtex#764] [Ask Anything]
![figure](https://user-images.githubusercontent.com/75380911/107546764-03bb3400-6bcd-11eb-9630-e17ea01c3c0b.jpg)

He Ralf,

I projected samples axes into the fundamental region of an FCC system and I get the figure in view. 
The upper corner of the triangle must be the conjugate plane (-111) (or direction) instead of (1-1-1).

Is it true?

Thank you in advance
Mohamed

**Comment by ralfHielscher** [comment]
Hi Mohamed,

to me this appears fine. Why do you think that this is wrong?

Ralf.

**Reply by M-Eddahbi** [reply]
Hi Ralf,
Thank you for the answer.
The notation [1-1-1] is wrong. The correct one is [-111] (See the stereographic projection in view)
![Proyeccion1](https://user-images.githubusercontent.com/75380911/107681907-f91d9f00-6c9f-11eb-9436-57848e5c7e14.jpg)


treographic projection of an FCC system reference).

**Comment by bjoerneske** [comment]
This is fine. I think you are overlooking a minus on top of the 00-1,
hidden under the big symbol? Then it is [00-1], [0-1-1], [1-1-1] , as in
the figure

tor. 11. feb. 2021, 19:34 skrev Eddahbi, Mohamed OuLahcen <
notifications@github.com>:

**Reply by M-Eddahbi** [reply]
Thank you Bjoernesk. But in this case it must be the (10-1) plane instead of (0-1-1)
![Both are wrong](https://user-images.githubusercontent.com/75380911/107702090-0cd5ff00-6cba-11eb-83da-07e877181060.jpg)
. (See the attached the figure)
The correct fundamental region in my case is (001), (011), (-111).
